slack_sdk/web/async_client.py

```diff
@@ -2075,12 +2075,15 @@ async def assistant_threads_setStatus(
         channel_id: str,
         thread_ts: str,
         status: str,
+        loading_messages: Optional[List[str]] = None,
         **kwargs,
     ) -> AsyncSlackResponse:
-        """Revokes a token.
```

Comment:
praise: Oof, thanks for catching this typo (and the ones following).

Reply:
No worries, since it's internal cleanup and the PR is small, it didn't distract from the core changes in the PR 😄 You made the right call!